server.js

```diff
@@ -24,6 +24,61 @@ app.get("/", (req, res) => {
   res.send("Hello Technigo!");
 });
 
+app.get("/books", (req, res) => {
+  const { title, authors } = req.query;
+  let allBooks = books;
+/*  Write for example: http://localhost:8080/books?authors=Gary Paulsen to get all the authors books
+    Write for example: http://localhost:8080/books?title=Hatchet to get the singel books info */
+
+  if (authors) {
+    allBooks = allBooks.filter(singleBook => singleBook.authors.toLowerCase() === authors.toLowerCase());
+  }
+
+  if (title) {
+    allBooks = allBooks.filter(singleBook => singleBook.title.toLowerCase() === title.toLowerCase());
+  } 
+
+  res.status(200).json({
+    success: true,
+    message: "OK",
+    body: { 
+      books: allBooks
+    }
+    });
+});
```

Comment:
The endpoints seem to work as intended. Good work Viktor. I also like that you made it easier for us to review your code by adding examples. Just a little suggestion for the comment part. If the name contains 2 words, you can add %20 between the word like this;

> /books?authors=Gary%20Paulsen